e2e/worker/worker_suite_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package worker_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/copilot-cli/e2e/internal/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	cli     *client.CLI
+	appName string
+)
+
+const (
+	lbwsServiceName   = "frontend"
+	workerServiceName = "worker"
+
+	envName = "test"
+)
+
+func TestWorker(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "App Runner Suite")
+}
+
+var _ = BeforeSuite(func() {
+	var err error
+	cli, err = client.NewCLI()
+	Expect(err).NotTo(HaveOccurred())
+	Expect(err).NotTo(HaveOccurred())
+	appName = fmt.Sprintf("e2e-worker-%d", time.Now().Unix())
+})
+
+var _ = AfterSuite(func() {
+	_, err := cli.SvcDelete(lbwsServiceName)
+	Expect(err).NotTo(HaveOccurred())
+
+	_, err = cli.SvcDelete(workerServiceName)
+	Expect(err).NotTo(HaveOccurred())
+
+	_, err = cli.EnvDelete(envName)
+	Expect(err).NotTo(HaveOccurred())
+
+	_, err = cli.AppDelete()
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
Why not just app delete? Is it because we have to remove them in a certain order?

Reply:
because i am stoopid and was just copy pasting without thinking. Done!